File: core/src/main/java/com/ustadmobile/core/view/RecordDropoutDialogView.java
package com.ustadmobile.core.view;


/**
 * Core View. Screen is for RecordDropoutDialog's View
 */
public interface RecordDropoutDialogView extends UstadView {


    // This defines the view name that is an argument value in the go() in impl.
    String VIEW_NAME = "RecordDropoutDialog";

    //Any argument keys:

    /**
     * Method to finish the screen / view.
     */
    void finish();


}

File: core/src/commonMain/kotlin/com/ustadmobile/core/view/RecordDropoutDialogView.kt
package com.ustadmobile.core.view


/**
 * Core View. Screen is for RecordDropoutDialog's View
 */
interface RecordDropoutDialogView : UstadView {

    //Any argument keys:

    /**
     * Method to finish the screen / view.
     */
    fun finish()

    companion object {


        // This defines the view name that is an argument value in the go() in impl.
        val VIEW_NAME = "RecordDropoutDialog"
    }


}

